packages/core/src/services/commits/create.ts

```diff
@@ -1,21 +1,23 @@
 import {
-  Commit,
   commits,
   Database,
   database,
-  Project,
   Result,
   Transaction,
 } from '@latitude-data/core'
+import { Commit, Project, SafeUser } from '$core/browser'
 
 export async function createCommit({
   project,
-  data: { title, mergedAt },
+  user,
+  data: { title, description, mergedAt },
   db = database,
 }: {
   project: Project
+  user: SafeUser
```

Comment:
what's the difference between user and safeuser?

Reply:
Safe user only exposes email,name,id.